esque/controller/topic_controller.py

```diff
@@ -8,8 +8,8 @@
 
```

Comment:
The `ConsumerGroupController` is in `esque.resources.consumergroup`. So either move that to `esque.controllers` or move this to `esque.resources.topic` to stay consistent. I think I'd prefer the former solution since it's closter to the MVC pattern which we're kind of using :smile:

Reply:
Fair. Move it.